nft_ingester/src/main.rs

```diff
@@ -38,7 +39,7 @@ async fn main() {
         .await
```

Comment:
We ought to move the Postgres connection params in line 39 into a configuration file so we can use unique Postgres databases in dev, staging, production.

Reply:
Created issue to track this: https://app.zenhub.com/workspaces/protocol-engineering-61eebb497fb4920012651e19/issues/metaplex-foundation/metaplex-program-library/538